tests/models/test_hf_model.py

```diff
@@ -104,6 +104,13 @@ def check_hf_tokenizer_equivalence(tokenizer1, tokenizer2):
     tokenizer1.__dict__.pop('deprecation_warnings')
     tokenizer2.__dict__.pop('deprecation_warnings')
 
+    # name or path will be the path that the tokenizer was loaded from, which will just be a temporary directory for
+    # the reloaded tokenizer
+    tokenizer1.__dict__.pop('name_or_path')
+    tokenizer2.__dict__.pop('name_or_path')
+    tokenizer1.init_kwargs.pop('name_or_path', None)
+    tokenizer2.init_kwargs.pop('name_or_path', None)
```

Comment:
I don't really understand this comment. Can you indicate where we re-set this var or why its not needed?
```suggestion
    # Name or path will be the path that the tokenizer was loaded from, which will just be a temporary directory for
    # the reloaded tokenizer
    tokenizer1.__dict__.pop('name_or_path')
    tokenizer2.__dict__.pop('name_or_path')
    tokenizer1.init_kwargs.pop('name_or_path', None)
    tokenizer2.init_kwargs.pop('name_or_path', None)
```
Nit: capitalize start of comment